overlay.d/05rhcos/usr/lib/dracut/modules.d/41rhcos-early-reboot/coreos-kargs-reboot.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=CoreOS Early Reboot
+ConditionPathExists=/etc/initrd-release
+DefaultDependencies=false
+Before=ignition-complete.target
+ 
+# This runs after ignition-kargs & before ignition-disks so that it can optionally reboot
+# if kargs were modified via Ignition. This is done in a two-stage fashion so that other
+# mechanisms which may want to reboot (e.x. FIPS) can also hook in here and only reboot
+# once from the initrd.
+After=ignition-kargs.service
+Before=ignition-disks.service
+
+# On RHCOS we also need to run after FIPS and reboot in the case that
```

Comment:
Offhand...I think nothing stops us from moving the FIPS bits into FCOS too.  It'd be an open question whether and how much we test it.  Definitely don't need to change it in this PR, just thinking out loud.